integration-cli/check_test.go

```diff
@@ -33,6 +33,7 @@ func (s *DockerSuite) TearDownTest(c *check.C) {
 	deleteAllImages()
 	deleteAllVolumes()
 	deleteAllNetworks()
+	cleanupShm()
```

Comment:
Couldn't figure out a way to do so. If the previous test crashes this is the only certain way to clean up behind that I found. As far as CI is concerned this could be safely removed though since we start from scratch every time. It just means that after a test failure there is no warranty that the test actually passes if it also uses shm.


Any suggestions?